build.sbt

```diff
@@ -1304,13 +1305,18 @@ lazy val `ydoc-server` = project
   .dependsOn(`logging-service-logback`)
   .dependsOn(`profiling-utils`)
 
+lazy val persistanceVersion = "0.2-SNAPSHOT"
```

Comment:
I'd prefer if this was at the top of the file, together with other hardcoded versions.